get-lease.yaml

```
@@ -0,0 +1,66 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: jumpstarter-get-lease
+  namespace: jumpstarter-pipelines
+spec:
+  params:
+    - default: ''
```

```suggestion
    - default: 'default'
```

default is the default name :)